docs/HowToAddAudioSupport.md

```diff
@@ -0,0 +1,62 @@
+
+# SECTION I: Tizen RT's Audio Framework
+
+Tizen RT's audio framework has 4 components spreading across different locations in code base:
+
+1. os/audio (Place holder for audio devices operations)
+2. os/drivers/audio (Place holder for audio codec drivers)
+3. framework/src/tinyalsa (Tinyalsa library)
+4. Depending on the embedded board, a control path (i2c, spi, etc) and a data path (i2s, etc).
+
+
+# SECTION II: How To Add Audio for a Board
+From SECTION I, changes for a new board are limited to #2 & #4. ie,   
+a. Adding control and data path  
+b. Adding BSP/firmware for audio codec  
+  
+Right now, Tizen RT supports alc5658 audio codec ONLY; through I2C (control path) & I2S (data path).
+
+# SECTION III: Control Path (I2C)
+
+1. os/include/tinyara/i2c.h has data structures and APIs of I2C
```

Comment:
1. Is this mentioned in audo doc? I think you had better assume there is a i2c driver and say how to connect i2c to audio.
2. Does all of audio hw use i2c as a control path?

Reply:
1. This is not a i2c driver. 
2. This document neve says that all audio hw use i2c as control path